scanner/redhatbase.go

```diff
@@ -19,8 +20,22 @@ import (
 // https://github.com/serverspec/specinfra/blob/master/lib/specinfra/helper/detect_os/redhat.rb
 func detectRedhat(c config.ServerInfo) (bool, osTypeInterface) {
 	if r := exec(c, "ls /etc/fedora-release", noSudo); r.isSuccess() {
-		logging.Log.Warnf("Fedora not tested yet: %s", r)
-		return true, &unknown{}
+		if r := exec(c, "cat /etc/fedora-release", noSudo); r.isSuccess() {
```

Comment:
I implemented to Fedora scanner as a group of `redhatbase`, because it is already written here.
Should it still be here?

Reply:
Fedora also has `/etc/redhat-release`, so it should be above the part that judges about redhat-release.
```console
[root@54c54d7d7d5d /]# cat /etc/redhat-release 
Fedora release 35 (Thirty Five)
```